setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-install_reqs = ["Django>=1.4", "sixpack-client==1.0.0"]
+install_reqs = ["Django>=1.4", "sixpack-client==1.1.1"]
```

Comment:
I'm thinking around migrations - if it's an old-style `south` migration, don't we need to add a requirement for `south`?

@ashchristopher, can you confirm how we should be handling migrations to work with both old and new? You've probably done more research into this than anyone else.